src/com/codefortomorrow/beginner/chapter2/practice/ApplesOranges.java

```diff
@@ -0,0 +1,28 @@
+package com.codefortomorrow.beginner.chapter2.practice;
+
+/**
+ * @author ArmeetJatyani
+ * March 2021
+ * 
+ * Print out the number of apples and oranges!
+ */
+
+ public class ApplesOranges {
+    public static void main(String[] args) {
+        // write your code here
+        
+        // define an integer variable called numOranges with value 10 (line 15)
+
+        
+        // define an integer variable called numApples with value 24 (line 18)
+
+        
+        // print out number of oranges, output: "I have 10 oranges." (line 21)
```

Comment:
```suggestion
        // print out number of oranges using variables, output: "I have 10 oranges." (line 21)
```
Just because some people might print out "I have 10 oranges" outright.